spec/controllers/spree/products_controller_spec.rb

```diff
@@ -8,7 +8,7 @@ module Spree
 
     context "visit non translated product page via permalink on url" do
       it "displays pages successfully" do
-        spree_get :show, id: product.slug, locale: 'pt-BR'
+        get :show, params: { id: product.slug, locale: 'pt-BR' }
```

Comment:
This will only work with Rails 5 and up, but we still want to support older versions of Solidus and therefore Rails. Please add the `rails_test_params_backport ` gem in order to support this kind of testing in Rails 4

Example, of how the `solidus_auth_devise` gets around this

https://github.com/solidusio/solidus_auth_devise/blob/master/Gemfile#L6-L12